src/mixins/audio_token_controls.js

```diff
@@ -91,28 +91,44 @@
       touchCornerSize: 64,
 
       cursorStyle: 'pointer',
+      mouseDownHandler: function (eventData, target) {
+        target.controls.playControl.heldDown = true;
+      },
       mouseUpHandler: function (eventData, target) {
         var canvas = target.canvas;
 
+        target.controls.playControl.heldDown = false;
         target.playControlPressed && target.playControlPressed(eventData);
         canvas.requestRenderAll();
       },
-
       render: function (ctx, left, top, styleOverride, fabricObject) {
         // fabricObject.scale is the 'base' scale, scaleX and scaleY are identical, and control the actual drawing scale
         var scale = fabricObject.scaleX;
         var size = this.cornerSize * scale;
         var controlImg;
 
         // draw either the default icons or the ones defined by parent (audio_token)
-        controlImg = fabricObject.isPlaying ?
-          fabricObject.pauseControlImage :
-          fabricObject.playControlImage;
+        if (fabricObject === fabricObject.canvas._indicatedObject) {
```

Comment:
Not sure what the Fabric convention is, but usually a variable starting with `_` is meant to be private. Is it the case here? Should we create an getter for it?

Reply:
@brassaro Usually yes, agree. I just followed the convention code in this case

Reply:
I believe @brassaro is correct here... it seems Fabric uses _ for private... and if you want to access these outside then getters are created. So I think if you want to follow the convention you should create a getter here.

Reply:
Oh, I think in this case I was lazy and was using this private vars in other places - so it's probably my fault if you saw this in other places. Can you add a getter in the canvas class for this?

Reply:
> Oh, I think in this case I was lazy and was using this private vars in other places - so it's probably my fault if you saw this in other places. Can you add a getter in the canvas class for this?

Ok